src/_help/enhance-documentation-content/markdown-support.md

```diff
@@ -159,6 +165,8 @@ paths:
   ...
 ```
 
-With files `docs/introduction.md`, `docs/getting-started.md`, `docs/use-cases.md` and `docs/use-cases-examples.md` right next to your contract document, you will be able to generate a comprehensive API documentation with nicely formatted content for your users.
```

Comment:
Can we remove the `x-topics` part of the examples here, to keep it simple? Keeping only the $ref: "docs/introductions.md" seems to be enought.

Reply:
I removed also the mention of topics at the end, as I wasn't able to find an appropriate way to introduce them here.